### PR TITLE
#174566170 Set up the development environment

### PR DESCRIPTION
**Pull request by Fiyin-Anne**
## What does this PR do?
This PR sets up the development environment by adding express and dotenv packages.

## Description of tasks to be completed?
- Install express package
- Install dotenv package
- set up the server to run on port 3000

## How should this be manually tested/checked?
- Clone the PR
- Cd into the branch `ch-setup-dev-env-174566170`
- Install the dependencies using `npm install`
- Run the server on the terminal using `npm run start`
- It can be viewed on this link: http://127.0.0.1:3000/

## What are the relevant pivotal tracker stories?
[#174566170](https://www.pivotaltracker.com/story/show/174566170)

### CODE REVIEW

**Comment by bellogo** [comment]
This meets the requirement to set development environment. well done Fiyin.

**Review by bellogo — APPROVED** [review]
This meets the requirement to set development environment. well done Fiyin.

**Review by codeliezel — COMMENTED** [review]
well done, so in order not to waste time @Fiyin-Anne  the 
- .env file should never be pushed and should be in gitignore

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+if (process.env.NODE_ENV !== 'production') {
```
Comment (codeliezel) [inline src/app.js:1]: remove this body of code, you do not need to set the environment for now.

**Review by codeliezel — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 1 above)
Comment (codeliezel) [reply src/app.js:1]: remove all and not only that line..

**Review by Bernard-Namangala — APPROVED** [review]

**Review by codeliezel — COMMENTED** [review]
Also, your PR header should look like this: **#167750041Set up the development environment**

So moving forward, that is how PR headers should be like

After, your PR comment should contain these things:

**What does this PR do?**

**Description of tasks to be completed?**

**How should this be manually tested/checked?**

**What are the relevant pivotal tracker stories?**
#167750045 // this should be a link to the PT story like <a href=""> #167750045 </a>

**Review by codeliezel — APPROVED** [review]

**Review by bellogo — APPROVED** [review]
I confirmed that the right changes were made

**Review by donaldcrane — APPROVED** [review]
nice work!

**Pull request merged**